packages/vats/src/localchain.js

```diff
@@ -100,9 +87,8 @@ export const LocalChainI = M.interface('LocalChain', {
   queryMany: M.callWhen(M.arrayOf(M.record())).returns(M.arrayOf(M.record())),
 });
 
-export const LocalChainAdminI = M.interface('LocalChainAdmin', {
-  setPower: M.callWhen(M.string(), M.await(M.any())).returns(),
-});
+// XXX vestigial? for future use?
```

Comment:
Looks like the motivation for a "kit" is gone. Is there good reason to expect it to return? If not, please simplify to a regular Exo.